### PR TITLE
7007: moves book description above omnibar

### LINKED ISSUE #7007: Move book description above omnibar
The description on book pages should be positioned directly above the omnibar in both mobile and desktop views.

Here is an example of how this should look on mobile devices:
Screenshot from 2022-09-26 19-38-21

Note that restyling the omnibar is out of scope for this issue.

### PR DESCRIPTION
**Pull request by jpdh9881**
Closes #7007 

feature

### Technical

- verified on Firefox + Chrome
- edit-toolbar styles: copied tablet breakpoint to a new desktop breakpoint and modified tablet breakpoint (to solve issue where edit-toolbar would share space with book description at tablet sizes)

### Testing

Look at work description on book page at all breakpoints

### Screenshot

Desktop
image
image

Tablet
image

Mobile
image

### Stakeholders

@jimchamp

### CODE REVIEW

**Comment by mekarpeles** [comment]
I think this is very close! @jimchamp do we want the description below the navigation bar?

**Comment by jimchamp** [comment]
Once all of the books page issues have been addressed, the navbar should be at the top of the page.  We may want to merge this and the other book page PRs into an integration branch, then merge that branch into main when enough of the issues have been addressed.

**Comment by jimchamp** [comment]
It looks like the edit component layout changes have affected many other pages negatively:
Screenshot from 2022-11-18 17-33-35
_Author edit page_

Screenshot from 2022-11-18 17-34-20
_Author page_

Screenshot from 2022-11-18 17-33-49
_Edit history page_

Suggest undoing all changes unrelated to the book description (`.book-description`) and the "Read More" buttons, and clearing the float on the book description for tablet views.

File: static/css/components/edit-toolbar--desktop.less [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+div#editInfo {
+  margin-top: 3px;
+  float: right;
+}
+
+div#editTools {
+  display: flex;
+  margin-top: 1px;
+  max-width: 280px;
+}
```
Comment (jimchamp) [inline static/css/components/edit-toolbar--desktop.less:24]:
```suggestion
```
Remove this file, as edit component layout changes are unnecessary.

File: static/css/components/edit-toolbar--tablet.less [hunk 2]
```diff
@@ -4,16 +4,21 @@
  */
 
 /* stylelint-disable selector-max-specificity */
+div.editionAll {
+  padding-bottom: 1rem;
+  display: flex;
+  justify-content: end;
+}
+
```
Comment (jimchamp) [inline static/css/components/edit-toolbar--tablet.less:12]:
```suggestion
```

File: static/css/components/edit-toolbar--tablet.less [hunk 3]
```diff
@@ -4,16 +4,21 @@
...
 div#editTools,
 div.editButton,
 div#editHistory {
+  display: flex;
```
Comment (jimchamp) [inline static/css/components/edit-toolbar--tablet.less:16]:
```suggestion
```
Remove `display: flex` and add `float: right` beneath the padding (revert to original styling).

File: static/css/components/edit-toolbar--tablet.less [hunk 4]
```diff
@@ -4,16 +4,21 @@
...
   padding: 3px 1px 0 20px;
-  float: right;
 }
 
 div#editInfo {
   margin-top: 3px;
-  float: right;
```
Comment (jimchamp) [inline static/css/components/edit-toolbar--tablet.less:16]: This float is needed.

File: static/css/components/work--tablet.less [hunk 5]
```diff
@@ -4,7 +4,6 @@
 // These elements are all children.
 div.edition {
   &Cover {
-    margin-bottom: 1.5em;
```
Comment (jimchamp) [inline static/css/components/work--tablet.less:7]: Don't remove this margin.

File: static/css/components/work--tablet.less [hunk 6]
```diff
@@ -4,7 +4,6 @@
...
     padding-right: 0;
   }
   &About {
```
Comment (jimchamp) [inline static/css/components/work--tablet.less:9]:
```suggestion
  &About {
    .book-description {
      clear: both;
    }
```
Adding the clear here will prevent the layout issue in tablet views.  Please add this below the `margin-top` rule on the next line.

File: static/css/page-book.less [hunk 7]
```diff
@@ -32,6 +32,10 @@
 @import (less) "components/buttonBtn.less";
 @import (less) "components/modal-links.less";
 
-@media only screen and (min-width: @width-breakpoint-tablet) {
+@media only screen and (min-width: @width-breakpoint-tablet) and (max-width: @width-breakpoint-desktop) {
```
Comment (jimchamp) [inline static/css/page-book.less:35]:
```suggestion
@media only screen and (min-width: @width-breakpoint-tablet) {
```

File: static/css/page-book.less [hunk 8]
```diff
@@ -32,6 +32,10 @@
...
   @import (less) "components/edit-toolbar--tablet.less";
 }
+
+@media only screen and (min-width: @width-breakpoint-desktop) {
+  @import (less) "components/edit-toolbar--desktop.less";
+}
```
Comment (jimchamp) [inline static/css/page-book.less:41]:
```suggestion
```

**Review by jimchamp — CHANGES_REQUESTED** [review]
Implementing the suggested changes should correct the layout issue for tablets without affecting other pages.

File: static/css/components/work.less [hunk 9]
```diff
@@ -375,10 +379,24 @@ div.editionTools {
...
+
 @media only screen and (max-width: @width-breakpoint-desktop){
   .desktop-book-header{
     display: none;
   }
+  div.editionAbout {
+    p.book-description {
+      text-align: center;
+    }
+  }
```
Comment (jimchamp) [inline static/css/components/work.less:399]:
```suggestion
  div.editionAbout {
    .book-description.read-more,
    .book-description.read-less {
      text-align: center;
    }
    .book-description {
      padding-top: 1em;
    }
  }
```

File: static/css/components/work.less [hunk 10]
```diff
@@ -375,10 +379,24 @@ div.editionTools {
...
 }
+
+@media only screen and (min-width: @width-breakpoint-tablet) and (max-width: @width-breakpoint-desktop){
+  div.editionAbout {
+    div.book-description {
+      padding-top: 1em;
+    }
+  }
+}
+
```
Comment (jimchamp) [inline static/css/components/work.less:390]:
```suggestion
```

**Comment by jimchamp** [comment]
Thanks @jpdh9881!  Made a small change to simplify the mobile CSS for the description and read more links.

Merging this into an integration branch that will contain our book page style changes.  Once enough changes are on that branch, we'll merge it into main.  If you are working on another book page issue, go ahead and pull from the integration branch (instead of `master`).

**Pull request merged**